chapters/arrays.tex

```diff
@@ -1380,6 +1383,19 @@ \subsection{Slice Operation}\label{slice-operation}
 but it is still possible to perform some operations on them.
 \end{example}
 
+\begin{example}
+General selection
+\begin{lstlisting}[language=modelica]
+  // Normal slice operation with extra array variable
+  Complex c[2]={Complex(1, 2), Complex(2, 3)}*Complex(1, 1);
```

Comment:
I find it strange to omit space around the binary operators when there is space after comma:
```suggestion
  Complex c[2] = {Complex(1, 2), Complex(2, 3)} * Complex(1, 1);
```